modules/webstyle/lib/invenio.wsgi

```diff
@@ -20,6 +20,10 @@
 mod_wsgi Invenio application loader.
 """
 
+# enable remote debuggin with wdb <https://github.com/Kozea/wdb>
+import os
+os.environ['WDB_NO_BROWSER_AUTO_OPEN'] = 'True'
+
```

Comment:
I would be in favour of enabling the debugger for the development site, not for production (so to move this under the `if CFG_DEVEL_SITE:`)